src/App.jsx

```diff
@@ -1,17 +1,27 @@
 import Navbar from "./components/Navbar";
 import GamesList from "./pages/GamesList";
 import Footer from "./components/Footer";
+
+import ProductDetailPage from "./pages/ProductDetailPage";
+
 import Toast from "./components/Toast";
 import "./App.css";
 
 
+
 function App() {
   return (
-    <div>
+    <div className="App">
       <Navbar />
+
+
+      <Footer />
+      <ProductDetailPage />
```

Comment:
Does this product details page only for one game? or for every game? because this is only render just one game